Yoast/javascript
# Add checks for words that look like they begin with a valid prefix

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds checks for stemming Indonesian words whose stem looks like it begins with a valid prefix. E.g. kelompok, membelanjakan (stem: belanja), keluyuran, meditasi.
* [Yoast SEO Premium] Improves word form recognition of Indonesian words whose stem looks like it begins with a valid prefix. E.g. percaya, kepercayaan, mempercayai, mempercayakan, kepercayaannya, percayakanlah, percayalah, percayakan.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

Check the unit tests:

* Run yarn test and make sure all tests still pass.
* Add more words whose stem looks like it begins with a valid prefix. E.g. keliling, merdeka, dialog, peduli, berani. Other example the words from this category can be found here https://github.com/Yoast/YoastSEO.js-premium-configuration/pull/195 and add the possible affixes. Example of possible affixes: **ber- + word, ke- + word + -an, mem- + word with be- beginning + [-kan/-i], di- + word + [-kan/-i], pe- + word + [-an]**  -> **[] is optional**

Check in the content-analysis app:

* Open the content analysis app by running yarn start;
* Insert a text of al least 150 words;
* Set the locale to Indonesian (i.e., id_ID);
* Make sure all of the following forms are being recognised as forms of the same word when one of them is entered in the keyphrase and others in the title/text/meta-description: berkeliling, pengeliling, keliling, pengelilingan, dikelilingi
* Make sure all of the following forms are being recognised as forms of the same word when one of them is entered in the keyphrase and others in the title/text/meta-description: merdeka, memerdekakan, dimerdekakan, pemerdekaan
* Make sure all of the following forms are being recognised as forms of the same word when one of them is entered in the keyphrase and others in the title/text/meta-description: peduli, kepedulian, memedulikan, pedulilah, pemeduli
* Make sure all of the following forms are being recognised as forms of the same word when one of them is entered in the keyphrase and others in the title/text/meta-description: berani, pemberani, memberanikan, pemberanian, keberanian, keberaniannya
* Check on the eye icon that all the word forms become highlighted

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-442
